frontend/src/Service/NeedAuthorization.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Redirect } from 'react-router';
+
+
+export function redirectToSignUp(error) {
+
+    /*this function checks error message for Forbidden cause.
+    If it present - return Redirect for LogIn page, else - return false*/
+
+    const isForbidden = ("" + error).toLowerCase().search("forbidden") !== -1;
```

Comment:
I'm not sure about this approach, it's unsafe as you cannot be sure that other errors will not contain "forbidden" word. You'd better add some field to response dict along with `success` and `error` which indicate unauthorized access, or use proper HTTP status code 403.